server/sql/dev/create-tables-dev.sql

```diff
@@ -1,18 +1,47 @@
 \c socobo_dev;
 
+--
+--  SOCOBO USER
+--
+
+CREATE TABLE Socobo_User_Role (
+  id SERIAL PRIMARY KEY,
+  name VARCHAR NOT NULL UNIQUE,
+  created NUMERIC NOT NULL,
+  lastModified NUMERIC NOT NULL CHECK (lastModified >= created)
+);
+
+CREATE TABLE Socobo_User_Provider (
+  id SERIAL PRIMARY KEY,
```

Comment:
hmm. i'm not really sure if i like strings as id's. we could return id and name as key value pairs to the frontend!?